src/modules/CircularGradient/circularGradient.js

```diff
@@ -0,0 +1,37 @@
+module.exports = exports = function(pixels, options) {
+
+    let defaults = require('./../../util/getDefaults.js')(require('./info.json'));
+
+    let fillColor = options.fillColor || defaults.fillColor;
```

Comment:
OK so here we can just set `fillColor` to white, so we can remove all the fillColor code, make sense? we only want monochrome for now.